shared/provision/code-page/container.tsx

```diff
@@ -26,13 +26,7 @@ export default Container.connect(
       dispatch(ProvisionGen.createSubmitTextCode({phrase: new HiddenString(code)})),
   }),
   (stateProps, dispatchProps, _: OwnProps) => ({
-    currentDeviceAlreadyProvisioned: stateProps.currentDeviceAlreadyProvisioned,
-    currentDeviceName: stateProps.currentDeviceName,
-    error: stateProps.error,
-    onBack: dispatchProps.onBack,
-    onSubmitTextCode: dispatchProps.onSubmitTextCode,
-    otherDeviceName: stateProps.otherDeviceName,
-    otherDeviceType: stateProps.otherDeviceType,
-    textCode: stateProps.textCode,
+    ...dispatchProps,
```

Comment:
Typically we've avoided doing these types of spreads as it's very easy to accidentally plumb extra props down and we currently can not detect this